packages/theme/components/Checkout/VsfShippingProvider.vue

```diff
@@ -43,13 +43,13 @@ export default {
     const selectedShippingRates = ref({});
 
     const allShipmentsSelected = computed(() => {
+      if (!shipments.value) return false;
       const shippingCodes = Object.values(selectedShippingRates.value);
-      return shippingCodes.length && shippingCodes.every(e => e !== null);
+      return shippingCodes.length === shipments.value.length && shippingCodes.every(e => e !== null);
```

Comment:
Where does this come from?

Reply:
I believe it works rather as a defensive programming check if any `ShippingRatePicker` would fail to call `@change` and then `selectShippingRate` afterwards, the button would stay disabled because the length of dynamically created `selectedShippingRates` values differs from the length of shipments returned from the API (in other words, not all shipments are selected [some would be actually missing]).

Additionally, when removing the former check, the `if (!shipments.value) return false;` is no longer needed, because `shippingCodes.length && ...` covers this case.